relay_handler.js

```diff
@@ -63,10 +63,14 @@ RelayHandler.prototype.handleLazily = function handleLazily(conn, reqFrame) {
     }
 
     if (self.circuits) {
-        self.logger.warn(
-            'circuit breaking for lazy relaying isn\'t implemented',
-            rereq.extendLogInfo({}));
-        return false;
+        var circuit = self.circuits.getCircuit(rereq.callerName, rereq.serviceName, rereq.endpoint);
```

Comment:
I don't really have a problem with them having separate methods. `getCircuitForRequest` makes sense since eager handling will have an *actual* request object. I could make the eager path also use `getCircuit`; that would just be moving a bit of code from `hyperbahn/circuits.js` to `relay_handler.js`. It's a simple refactor that I don't think is necessary for this PR.